lib/saml_idp/controller.rb

```diff
@@ -33,9 +33,9 @@ def acs_url
       end.new(nil)
     end
 
-    def validate_saml_request(raw_saml_request = params[:SAMLRequest])
+    def validate_saml_request(raw_saml_request = params[:SAMLRequest], sign_info = {})
       decode_request(raw_saml_request)
```

Comment:
Does it seem like we could pin this into the request? It seems odd that we have default access to a lot of the data we need off of the saml-request then we pass around signature info through a bunch of methods. 

Genuine question if makes sense or not :-) Conceptually it seems like a valid location for it.